tests/tracer.go

```diff
@@ -0,0 +1,57 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"os/signal"
+
+	"github.com/weaveworks/tcptracer-bpf/pkg/event"
+	"github.com/weaveworks/tcptracer-bpf/pkg/tracer"
```

Comment:
Shouldn't all the top-level functions be available from the top-level package? (i.e. `github.com/weaveworks/tcptracer-bpf/` directly)